lib/membrane_rtmp_plugin/rtmp/sink/sink.ex

```diff
@@ -40,34 +42,40 @@ defmodule Membrane.RTMP.Sink do
                 description:
                   "Destination URL of the stream. It needs to start with rtmp:// or rtmps:// depending on the protocol variant.
                 This URL should be provided by your streaming service."
+              ],
+              max_attempts: [
+                type: :integer,
+                spec: pos_integer(),
+                default: 1,
```

Comment:
I'm wondering if it shouldn't be set to a higher value 🤔

Reply:
I wanted to keep the default behaviour. Besides, it won't always be needed and I think making the retry mechanism should stay optional.